stretch_moveit_config/config/ros_controllers.yaml

```diff
@@ -1,31 +1,30 @@
-# Publish all joint states
-# Creates the /joint_states topic necessary in ROS
-joint_state_controller:
-  type: joint_state_controller/JointStateController
-  publish_rate: 50
-controller_list:
-  - name: stretch_arm_controller
-    action_ns: follow_joint_trajectory
-    default: True
-    type: FollowJointTrajectory
+controller_manager:
+  ros__parameters:
+    update_rate: 100  # Hz
+
+    stretch_controller:
+      type: joint_trajectory_controller/JointTrajectoryController
+
+    joint_state_broadcaster:
+      type: joint_state_broadcaster/JointStateBroadcaster
```

Comment:
Seems like stretch_controller and joint_state_broadcaster aren't getting loaded correctly. Getting the following error when starting the `demo.launch.py` launch file:

```
[ros2_control_node-5] [INFO] [1634457202.898422014] [controller_manager]: Loading controller 'stretch_controller'
[ros2_control_node-5] [ERROR] [1634457202.898496687] [controller_manager]: Available classes:
[ros2_control_node-5] [ERROR] [1634457202.898543240] [controller_manager]: controller_manager/test_controller
[ros2_control_node-5] [ERROR] [1634457202.898562555] [controller_manager]: controller_manager/test_controller_failed_init
[ros2_control_node-5] [ERROR] [1634457202.898576733] [controller_manager]: controller_manager/test_controller_with_interfaces
[ros2_control_node-5] [ERROR] [1634457202.898592582] [controller_manager]: Loader for controller 'stretch_controller' not found.
[ros2_control_node-5] [INFO] [1634457202.955350521] [controller_manager]: Loading controller 'joint_state_broadcaster'
[ros2_control_node-5] [ERROR] [1634457202.955463564] [controller_manager]: Available classes:
[ros2_control_node-5] [ERROR] [1634457202.955496096] [controller_manager]: controller_manager/test_controller
[ros2_control_node-5] [ERROR] [1634457202.955513931] [controller_manager]: controller_manager/test_controller_failed_init
[ros2_control_node-5] [ERROR] [1634457202.955534576] [controller_manager]: controller_manager/test_controller_with_interfaces
[ros2_control_node-5] [ERROR] [1634457202.955555342] [controller_manager]: Loader for controller 'joint_state_broadcaster' not found.
```

Reply:
More relevant errors ([full log here](https://gist.github.com/hello-binit/d10ce29c4a8d7250481267e3f824234e)):

```
[ERROR] [spawner-6]: process has died [pid 57799, exit code 1, cmd '/home/hello-binit/ament_ws/install/controller_manager/lib/controller_manager/spawner stretch_controller --ros-args'].
[ERROR] [spawner-7]: process has died [pid 57801, exit code 1, cmd '/home/hello-binit/ament_ws/install/controller_manager/lib/controller_manager/spawner joint_state_broadcaster --ros-args'].
```